documentation/general/dash-sonic-hld.md

```diff
@@ -494,8 +495,25 @@ vm_vni                   = VM VNI that is used for setting direction. Also used
 
 ### 3.2.9 ROUTE LPM TABLE - OUTBOUND
 
+```
+DASH_ENI_ROUTE_TABLE:{{eni}}:
+    "group_id": {{group_id}} 
+```
+
+```
+key                      = DASH_ENI_ROUTE_TABLE:eni ; ENI name as key; ENI can bind to a route group table atomically (Overwrites any old bindings)
+; field                  = value 
+group_id                 = Route Group Table Id (contains both IPv4 and IPv6 routes)
+```
+
+```
+DASH_ROUTE_GROUP_TABLE:{{group_id}}
+    "guid": {{string}}
```

Comment:
do we still need the guid here? @sharmasushant

Reply:
guid can be optional

Reply:
I don't see a use for guid field, but I am okay to leave it there.